bolt-design-system/bolt
# Fix: icon display issue in Safari

## Jira

http://vjira2:8080/browse/BDS-712

## Summary

Fixes an issue with certain icons not rendering in Safari browsers.

## Details

Certain icons were not appearing in Safari browsers due to the masking within the SVG files. I re-saved those icons without the masking and flatten them.

## How to test

Run the branch locally, load up the icon demo page (http://localhost:3000/pattern-lab/?p=components-icon-name-variations) in Mac Safari and iOS Safari (using browserstack or xcode simulator). Make sure all icons render on the page.
